shuup/apps/settings.py

```diff
@@ -119,3 +121,23 @@ def reload_apps():
     django.apps.apps.app_configs.clear()
     django.apps.apps.ready = False
     django.setup()
+
+
+def get_configuration_setting(setting_key: str):
+    from shuup import configuration
+
+    modules = [
+        "shuup.core.setting_keys",
```

Comment:
`shuup/apps` shouldn't know anything abount `shuup/core` or shuup/admin`. Can you create some provide key to load them dynamically?